exporter/signalfxexporter/translation/constants.go

```diff
@@ -138,6 +138,37 @@ translation_rules:
     receive: pod_network_receive_errors_total
     transmit: pod_network_transmit_errors_total
 
+# cpu deltas
+- action: delta_metric
+  mapping:
+    system.cpu.time: system.cpu.delta
+- action: copy_metrics
+  mapping:
+    system.cpu.delta: system.cpu.usage
+  dimension_key: state
+  dimension_values:
+    user: true
+    system: true
+    interrupt: true
```

Comment:
there must be more states to be aggregated except for idle:
interrupt
nice
softirq
steal
system
user
wait

Reply:
Done.